config/authorities/linked_data/loc.json

```diff
@@ -31,7 +31,9 @@
       "id_predicate": "http://id.loc.gov/vocabulary/identifiers/lccn",
       "label_predicate": "http://www.w3.org/2004/02/skos/core#prefLabel",
       "altlabel_predicate": "http://www.w3.org/2004/02/skos/core#altLabel",
-      "sameas_predicate": "http://www.w3.org/2004/02/skos/core#exactMatch"
+      "sameas_predicate": "http://www.w3.org/2004/02/skos/core#exactMatch",
+      "narrower_predicate": "http://www.loc.gov/mads/rdf/v1#hasNarrowerAuthority",
+      "broader_predicate": "http://www.loc.gov/mads/rdf/v1#hasBroaderAuthority"
```

Comment:
Some LOC authorities define narrower and broader, so they should be included hear to be available for normalization.